.snyk

```diff
@@ -50,62 +38,6 @@ ignore:
     - uswds > del > globby > fast-glob > glob-parent:
         reason: Created issue to resolve in 90-days
         expires: '2021-04-12T21:40:40.202Z'
-  SNYK-JS-NODESASS-1059081:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-04-12T21:40:40.202Z'
-  SNYK-JS-NODESASS-535498:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-04-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-535500:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 30-days
-      expires: '2021-03-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-535502:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-05-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-540956:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 30-days
-      expires: '2021-03-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-540958:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-05-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-540990:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-05-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-540996:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 30-days
-      expires: '2021-03-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-540998:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-05-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-540992:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-05-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-540994:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-05-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-540964:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-05-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-540978:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-05-14T21:40:40.202Z'
-  SNYK-JS-NODESASS-540980:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-05-14T21:40:40.202Z'
 # patches apply the minimum changes required to fix a vulnerability
```

Comment:
You should remove these patches. I'm not seeing `snyk patch` in the package.json scripts, so these could be hiding true positives.